src/catalog/category-product-rules.md

```diff
@@ -38,14 +38,22 @@ Use caution when applying a category product rule, because any products that do
 
 1. Click <span class="btn">Add Condition</span> and do the following:
```

Comment:
If you convert bullet items to steps at same level, you have to remove this part

```suggestion
1. Click <span class="btn">Add Condition</span>.
```